config/cesm/config_grids.xml

```diff
@@ -120,6 +120,15 @@
       <grid name="rof">null</grid>
     </model_grid>
 
+    <!-- Regional NLDAS grid over the U.S. (roughly 12km resolution), with the standard NLDAS mask -->
+    <model_grid alias="nldas_mnldas" compset="DATM.+CLM">
+      <grid name="atm">224x464_nldas</grid>
+      <grid name="lnd">224x464_nldas</grid>
+      <grid name="ocnice">224x464_nldas</grid>
+      <grid name="rof">null</grid>
```

Comment:
I'm using the same mechanism we have used elsewhere to turn off ROF for this regional grid: setting the rof grid to null.